### PR TITLE
Add task for temp_dir creation

### PR DESCRIPTION
**Pull request by klyonrad**
fixes the broken pull request https://github.com/dkdeploy/dkdeploy-core/pull/8

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Added
 - tasks around MySQL slow log
+- add utils:create_local_temp_directory task
```
Comment (websi) [inline CHANGELOG.md]: Please use code block for capistrano tasks. See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting

File: lib/dkdeploy/tasks/mysql.rake [hunk 2]
```diff
@@ -25,7 +25,7 @@ namespace :mysql do
       next unless slow_log_exists? mysql_slow_log
       local_filename = File.join(local_dump_path, "#{File.basename(mysql_slow_log, '.*')}.#{fetch(:stage)}.#{server.hostname}#{File.extname(mysql_slow_log)}")
       info I18n.t('file.download', file: mysql_slow_log, target: local_filename, host: server)
-      execute :mkdir, '-p', local_dump_path
+      invoke 'utils:create_local_temp_directory'
```
Comment (websi) [inline lib/dkdeploy/tasks/mysql.rake]: Please define task as task dependency

File: lib/dkdeploy/tasks/mysql.rake [hunk 3]
```diff
@@ -42,7 +42,7 @@ namespace :mysql do
       execute :rm, '-f', remote_filename
       info I18n.t('tasks.mysql.analyze_slow_log', host: server, scope: :dkdeploy)
       execute :mysqldumpslow, '-s', 't', mysql_slow_log, '>', remote_filename
-      execute :mkdir, '-p', local_dump_path
+      invoke 'utils:create_local_temp_directory'
```
Comment (websi) [inline lib/dkdeploy/tasks/mysql.rake]: Please define task as task dependency

File: lib/dkdeploy/tasks/utils.rake [hunk 4]
```diff
@@ -138,4 +138,9 @@ namespace :utils do
       end
     end
   end
+
+  desc 'Create local temporary directory'
+  task :create_local_temp_directory do
+    run_locally { FileUtils.mkdir_p local_dump_path }
```
Comment (websi) [inline lib/dkdeploy/tasks/utils.rake]: Please use method `execute` to create local directory

**Review by websi — CHANGES_REQUESTED** [review]

**Review by websi — CHANGES_REQUESTED** [review]
Some test fails:

```
Failing Scenarios:
cucumber features/mysql.feature:8 # Scenario: Downloading the MYSQL slow log
cucumber features/mysql.feature:15 # Scenario: Downloading the MYSQL slow log analyze file
cucumber features/mysql.feature:22 # Scenario: Clearing the MySQL slow log file
```

**Comment by klyonrad** [comment]
@websi _works on my machine_

tested with ruby 2.2.5 and 2.1.9
gem versions for 2.2.5 (bundler 1.12.5):

```
bundle
Resolving dependencies...
Using rake 11.2.2
Using net-ssh 3.2.0
Using ffi 1.9.14
Using contracts 0.14.0
Using builder 3.2.2
Using gherkin 4.0.0
Using cucumber-wire 0.0.1
Using diff-lcs 1.2.5
Using multi_json 1.12.1
Using multi_test 0.1.2
Using rspec-support 3.5.0
Using thor 0.19.1
Using ast 2.3.0
Using bundler 1.12.5
Using capistrano-harrow 0.5.3
Using i18n 0.7.0
Using coderay 1.1.1
Using highline 1.7.8
Using mysql2 0.3.21
Using method_source 0.8.2
Using slop 3.6.0
Using powerpack 0.1.1
Using rainbow 2.1.0
Using ruby-progressbar 1.8.1
Using unicode-display_width 1.1.1
Using net-scp 1.2.1
Using childprocess 0.5.9
Using cucumber-core 1.5.0
Using rspec-expectations 3.5.0
Using rspec-core 3.5.3
Using rspec-mocks 3.5.0
Using parser 2.3.1.2
Using pry 0.10.4
Using sshkit 1.10.0
Using cucumber 2.4.0
Using rspec 3.5.0
Using rubocop 0.42.0
Using airbrussh 1.1.1
Using aruba 0.14.2
Using capistrano 3.5.0
Using dkdeploy-test_environment 1.0.0
Using dkdeploy-core 8.0.0 from source at `.`
```

**Comment by websi** [comment]
Not on my machine.
Error Message:

```
    And I successfully run `cap dev mysql:clear_slow_log`                                                               # aruba-0.14.1/lib/aruba/cucumber/command.rb:27
      expected "cap dev mysql:clear_slow_log" to be successfully executed (RSpec::Expectations::ExpectationNotMetError)
      features/mysql.feature:26:in `And I successfully run `cap dev mysql:clear_slow_log`'
```

Environment:

```
ruby 2.1.8p440 (2015-12-16 revision 53160) [x86_64-darwin14.0]

Using rake 11.2.2
Using net-ssh 3.2.0
Using ffi 1.9.10
Using contracts 0.14.0
Using builder 3.2.2
Using gherkin 4.0.0
Using cucumber-wire 0.0.1
Using diff-lcs 1.2.5
Using multi_json 1.12.1
Using multi_test 0.1.2
Using rspec-support 3.4.1
Using thor 0.19.1
Using ast 2.3.0
Using bundler 1.12.5
Using capistrano-harrow 0.5.2
Using i18n 0.7.0
Using coderay 1.1.1
Using highline 1.7.8
Using mysql2 0.3.21
Using method_source 0.8.2
Using slop 3.6.0
Using powerpack 0.1.1
Using rainbow 2.1.0
Using ruby-progressbar 1.8.1
Using unicode-display_width 1.0.5
Using net-scp 1.2.1
Using childprocess 0.5.9
Using cucumber-core 1.5.0
Using rspec-expectations 3.4.0
Using rspec-core 3.4.4
Using rspec-mocks 3.4.1
Using parser 2.3.1.2
Using pry 0.10.3
Using sshkit 1.10.0
Using cucumber 2.4.0
Using rspec 3.4.0
Using rubocop 0.40.0
Using airbrussh 1.0.2
Using aruba 0.14.1
Using capistrano 3.5.0
Using dkdeploy-test_environment 1.0.0
Using dkdeploy-core 8.0.0 from source at `.`
```

**Review by websi — APPROVED** [review]

**Pull request merged**